doc/c-backend/examples/latest.c

```diff
@@ -0,0 +1,82 @@
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef int64_t date_t;
+
+// gen$date = DATE
+struct icicle_state {
+    date_t      gen_date;
+    bool        acc_conv_6_set;
+    int64_t     acc_conv_6_val;
+
+    int64_t     new_count;
+    int64_t    *new_fact;
+    date_t     *new_date;
+
+    int64_t     output_repl;
+
+    const char *error;
+};
+
+void compute(struct icicle_state *s)
+{
+    // init [Mutable] [Option Int] acc$conv$6 = None : Option Int;
+    bool    acc_conv_6_set = false;
+    int64_t acc_conv_6_val = 0;
```

Comment:
ah, I see what you meant when you said options needed to be converted to (bool, val) in Avalanche now. Doing it in Core would be easier and still makes sense, I think....